src/common/impl_list_item.hpp

```diff
@@ -87,29 +89,23 @@ struct impl_list_item_t {
         : public type_deduction_helper_t<pd_t> {};
 
     template <typename pd_t>
-    impl_list_item_t(type_deduction_helper_t<pd_t>) {
-        using deduced_pd_t = typename type_deduction_helper_t<pd_t>::type;
-        create_pd_func_ = &primitive_desc_t::create<deduced_pd_t>;
+    constexpr impl_list_item_t(type_deduction_helper_t<pd_t>)
+        : create_pd_func_(&primitive_desc_t::create<typename type_deduction_helper_t<pd_t>::type>) {
```

Comment:
clang-format check failed in CI. Please apply clang-format to the modified files: [coding-style](https://github.com/oneapi-src/oneDNN/blob/master/CODING_STANDARDS.md#coding-style)

Reply:
Sorry, no. As far as I can tell after running .github/automation/clang-format.sh, the lines that the CI is complaining about are not lines I touched. Therefore, the problem existed before I touched the code.

Reply:
As I can see the [check ](https://dev.azure.com/mkldnn/dnnl/_build/results?buildId=828&view=logs&j=0bf7d65c-1444-5302-7319-c6947569d8d5&t=85a4fd38-810c-57ff-5840-0f66a7da7b77)complains about two files that were touched by this PR:
* modified:   src/common/impl_list_item.hpp
* modified:   src/common/utils.hpp

The check failed because you introduced too long lines.
Diff after applying clang-format is below. Please fix it.
```patch
diff --git a/src/common/impl_list_item.hpp b/src/common/impl_list_item.hpp
index 21de27500..e7833f199 100644
--- a/src/common/impl_list_item.hpp
+++ b/src/common/impl_list_item.hpp
@@ -90,13 +90,13 @@ struct impl_list_item_t {

     template <typename pd_t>
     constexpr impl_list_item_t(type_deduction_helper_t<pd_t>)
-        : create_pd_func_(&primitive_desc_t::create<typename type_deduction_helper_t<pd_t>::type>) {
-    }
+        : create_pd_func_(&primitive_desc_t::create<
+                          typename type_deduction_helper_t<pd_t>::type>) {}

     template <typename pd_t>
     constexpr impl_list_item_t(concat_type_deduction_helper_t<pd_t>)
-        : create_concat_pd_func_(concat_type_deduction_helper_t<pd_t>::type::create) {
-    }
+        : create_concat_pd_func_(
+                concat_type_deduction_helper_t<pd_t>::type::create) {}

     template <typename pd_t>
     constexpr impl_list_item_t(sum_type_deduction_helper_t<pd_t>)
@@ -105,8 +105,8 @@ struct impl_list_item_t {

     template <typename pd_t>
     constexpr impl_list_item_t(reorder_type_deduction_helper_t<pd_t>)
-        : create_reorder_pd_func_(reorder_type_deduction_helper_t<pd_t>::type::create) {
-    }
+        : create_reorder_pd_func_(
+                reorder_type_deduction_helper_t<pd_t>::type::create) {}

     explicit operator bool() const {
         return !utils::everyone_is(nullptr, create_pd_func_,
diff --git a/src/common/utils.hpp b/src/common/utils.hpp
index 1a789cda7..fe4f50c5e 100644
--- a/src/common/utils.hpp
+++ b/src/common/utils.hpp
@@ -624,7 +624,7 @@ private:
     bool initialized_;

 public:
-    constexpr setting_t() : value_{}, initialized_ {false} {}
+    constexpr setting_t() : value_ {}, initialized_ {false} {}
     constexpr setting_t(const T init) : value_ {init}, initialized_ {false} {}
     bool initialized() { return initialized_; }
     T get() { return value_; }
```

Reply:
Done